NDTensors/test/blocksparse.jl

```diff
@@ -45,26 +46,29 @@ using Test
     @test nnzblocks(A) == 2
     @test nnzblocks(B) == 2
 
-    A[1, 5] = 15
-    A[2, 5] = 25
-
-    @test A[1, 1] == 0
-    @test A[1, 5] == 15
-    @test A[2, 5] == 25
+    GPUArraysCore.@allowscalar begin
```

Comment:
Same comment about 
```suggestion
    @allowscalar begin
```

Reply:
I have removed this from all files now they all read `@allowscalar`